src/config/settings.js

```diff
@@ -8,27 +8,38 @@ const fs = require('fs');
 const util = require('util');
 const { dirname } = require('path');
 const deepmerge = require('deepmerge');
+const deepequal = require('deep-equal')
```

Comment:
Shouldn't that get listed in `packages.json` then? (I assume it's being pulled in via recursive dependencies now; but we should mark it as an explicit one in case it is no longer required by whatever requires it now.)

Reply:
Done, but with lodash.

Reply:
got the commit wrong, and my build is broken. stay tuned.